fenix/app/src/main/res/layout/fragment_generate_summary_with_ai.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:layout_margin="24dp">
+
+    <TextView
```

Comment:
I considered to build the UI using Compose, however, for the sake of maintaining a consistency in this module, I decided to respect the existing implementation. In a production code, I would bring this point to the team and understand all the constraints we would experience, such as time, deadline, workforce and motivations.

Reply:
[Jetpack Compose code built by me in the past](https://github.com/ivamacio/bookmyroom).